<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AI_InstantFrame</class>
 <widget class="QDialog" name="AI_InstantFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>762</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>762</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AI_Instant</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lblYCoordinateMax">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblYCoordinateMid">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblYCoordinateMin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>362</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-10.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="graphFrame">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>39</y>
     <width>660</width>
     <height>340</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblXCoordinateStart">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0Sec</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblXCoordinateEnd">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>380</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>10Sec</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>400</y>
     <width>478</width>
     <height>51</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblInterval">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>402</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample Interval:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtTimeValue">
   <property name="geometry">
    <rect>
     <x>638</x>
     <y>400</y>
     <width>45</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblTimeUnit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>400</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="sldTimerValue">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>427</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfigure">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>480</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>547</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>633</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnConfigure</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnPause</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>sldTimerValue</tabstop>
  <tabstop>edtTimeValue</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources>
  <include location="ai_instant.qrc"/>
 </resources>
 <connections/>
</ui>
